clevertap-core/src/main/java/com/clevertap/android/sdk/CleverTapAPI.java

```diff
@@ -1619,19 +1627,60 @@ public int getCount(String event) {
         return -1;
     }
 
+    /**
+     * Retrieves the count of logged events for a specific event name associated with the current user/{@link CleverTapAPI#getCleverTapID(OnInitCleverTapIDListener) CleverTap ID}.
+     * This operation involves a database query and should be called from a background thread.
+     * <br>
+     * Example usage:
+     * <br>
+     * <code>
+     * // Call from background thread <br>
+     * int itemSelectedCount = getUserEventLogCount("item_selected")
+     * </code>
+     *
+     * @param eventName Name of the event to get the count for (e.g., "navigation_clicked", "item_selected")
+     * @return The number of times the specified event has occurred for current user, or -1 if the event does not exist or there was an error
```

Comment:
Why do we return -1 if there are no records? Shouldn't we return 0 if there are no records and return -1 only if there is an error?

Reply:
resolved by @vasct